vulnerabilities/lib_oval.py

```diff
@@ -39,29 +39,29 @@
 
 1. Create an OvalDocument:
 
-    >>> tree = ElementTree()
-    >>> tree.parse("OvalTest.xml")
-    >>> document = OvalDocument(tree)
+    >> tree = ElementTree()
+    >> tree.parse("OvalTest.xml")
+    >> document = OvalDocument(tree)
 
 2. Find an oval element within the loaded document:
 
-    >>> element = document.getElementByID("oval:org.mitre.oval:def:22382")
-    >>> if element is not None:
-    >>>    ....
+    >> element = document.getElementByID("oval:org.mitre.oval:def:22382")
+    >> if element is not None:
+    >>    ....
 
 3. Read an XML file with a single OVAL Definition (error checking omitted for brevity):
 
-    >>> tree = ElementTree()    
-    >>> tree.parse('test-definition.xml')
-    >>> root = tree.getroot()    
-    >>> definition = lib_oval.OvalDefinition(root)
+    >> tree = ElementTree()    
+    >> tree.parse('test-definition.xml')
+    >> root = tree.getroot()    
+    >> definition = lib_oval.OvalDefinition(root)
     
 4. Change information in the definition from #3 and write the changes
 
-    >>> meta = definition.getMetadata()
-    >>> repo = meta.getOvalRepositoryInformation()
-    >>> repo.setMinimumSchemaVersion("5.9")
-    >>> tree.write("outfilename.xml", UTF-8", True)
+    >> meta = definition.getMetadata()
```

Comment:
I don't have any preferences here, either of them is fine by me. Is there a reason you go with `>>` ?.

Reply:
Yes this is to avoid pytest treating these as real doctests and failing since these are not meant to be passing code